pkg/virt-launcher/virtwrap/live-migration-source.go

```diff
@@ -855,6 +867,17 @@ func (m *migrationMonitor) startMonitor() {
 	}
 }
 
+// logMigrationInfo logs the same migration info as `virsh -r domjobinfo`
+func logMigrationInfo(logger *log.FilteredLogger, uid string, info *libvirt.DomainJobInfo) {
+	logger.V(4).Info(fmt.Sprintf(`Migration info for %s: TimeElapsed:%dms DataProcessed:%dB DataRemaining:%dB DataTotal:%dB `+
+		`MemoryProcessed:%dB MemoryRemaining:%dB MemoryTotal:%dB MemoryBandwidth:%dB/s DirtyRate:%dpages/s PageSize:%dB `+
+		`Iteration:%d PostcopyRequests:%d ConstantPages:%d NormalPages:%d NormalData:%dB ExpectedDowntime:%dms`,
+		uid, info.TimeElapsed, info.DataProcessed, info.DataRemaining, info.DataTotal,
+		info.MemProcessed, info.MemRemaining, info.MemTotal, info.MemBps, info.MemDirtyRate, info.MemPageSize,
```

Comment:
@jean-edouard Would it make sense to just present the dirty rate in bytes, info.MemDirtyRate *  info.MemPageSize as a single number?

Also, perhaps we should also print the diskBps?

Reply:
Thanks!